lib/node_modules/@stdlib/ndarray/base/mskfilter/lib/10d.js

```diff
@@ -0,0 +1,265 @@
+	var xbuf;
+	var mbuf;
+	var ybuf;
+	var dx0;
+	var dx1;
+	var dx2;
+	var dx3;
+	var dx4;
+	var dx5;
+	var dx6;
+	var dx7;
+	var dx8;
+	var dx9;
+	var dm0;
+	var dm1;
+	var dm2;
+	var dm3;
+	var dm4;
+	var dm5;
+	var dm6;
+	var dm7;
+	var dm8;
+	var dm9;
+	var dy0;
+	var S0;
+	var S1;
+	var S2;
+	var S3;
+	var S4;
+	var S5;
+	var S6;
+	var S7;
+	var S8;
+	var S9;
+	var sh;
+	var sm;
+	var sx;
+	var ix;
+	var im;
+	var iy;
+	var i0;
+	var i1;
+	var i2;
+	var i3;
+	var i4;
+	var i5;
+	var i6;
+	var i7;
+	var i8;
+	var i9;
+
+	// Extract loop variables: dimensions and loop offset (pointer) increments...
+	sh = x.shape;
+	sx = x.strides;
+	sm = mask.strides;
+	if ( x.order === 'row-major' ) {
+		// For row-major ndarrays, the last dimensions have the fastest changing indices...
+		S0 = sh[ 9 ];
+		S1 = sh[ 8 ];
+		S2 = sh[ 7 ];
+		S3 = sh[ 6 ];
+		S4 = sh[ 5 ];
+		S5 = sh[ 4 ];
+		S6 = sh[ 3 ];
+		S7 = sh[ 2 ];
+		S8 = sh[ 1 ];
+		S9 = sh[ 0 ];
+		dx0 = sx[ 9 ];                // offset increment for innermost loop
+		dx1 = sx[ 8 ] - ( S0*sx[9] );
+		dx2 = sx[ 7 ] - ( S1*sx[8] );
+		dx3 = sx[ 6 ] - ( S2*sx[7] );
+		dx4 = sx[ 5 ] - ( S3*sx[6] );
+		dx5 = sx[ 4 ] - ( S4*sx[5] );
+		dx6 = sx[ 3 ] - ( S5*sx[4] );
+		dx7 = sx[ 2 ] - ( S6*sx[3] );
+		dx8 = sx[ 1 ] - ( S7*sx[2] );
+		dx9 = sx[ 0 ] - ( S8*sx[1] ); // offset increment for outermost loop
+		dm0 = sm[ 9 ];
+		dm1 = sm[ 8 ] - ( S0*sm[9] );
+		dm2 = sm[ 7 ] - ( S1*sm[8] );
+		dm3 = sm[ 6 ] - ( S2*sm[7] );
+		dm4 = sm[ 5 ] - ( S3*sm[6] );
+		dm5 = sm[ 4 ] - ( S4*sm[5] );
+		dm6 = sm[ 3 ] - ( S5*sm[4] );
+		dm7 = sm[ 2 ] - ( S6*sm[3] );
+		dm8 = sm[ 1 ] - ( S7*sm[2] );
+		dm9 = sm[ 0 ] - ( S8*sm[1] );
+	} else { // order === 'column-major'
+		// For column-major ndarrays, the first dimensions have the fastest changing indices...
+		S0 = sh[ 0 ];
+		S1 = sh[ 1 ];
+		S2 = sh[ 2 ];
+		S3 = sh[ 3 ];
+		S4 = sh[ 4 ];
+		S5 = sh[ 5 ];
+		S6 = sh[ 6 ];
+		S7 = sh[ 7 ];
+		S8 = sh[ 8 ];
+		S9 = sh[ 9 ];
+		dx0 = sx[ 0 ];                // offset increment for innermost loop
+		dx1 = sx[ 1 ] - ( S0*sx[0] );
+		dx2 = sx[ 2 ] - ( S1*sx[1] );
+		dx3 = sx[ 3 ] - ( S2*sx[2] );
+		dx4 = sx[ 4 ] - ( S3*sx[3] );
+		dx5 = sx[ 5 ] - ( S4*sx[4] );
+		dx6 = sx[ 6 ] - ( S5*sx[5] );
+		dx7 = sx[ 7 ] - ( S6*sx[6] );
+		dx8 = sx[ 8 ] - ( S7*sx[7] );
+		dx9 = sx[ 9 ] - ( S8*sx[8] ); // offset increment for outermost loop
+		dm0 = sm[ 0 ];
+		dm1 = sm[ 1 ] - ( S0*sm[0] );
+		dm2 = sm[ 2 ] - ( S1*sm[1] );
+		dm3 = sm[ 3 ] - ( S2*sm[2] );
+		dm4 = sm[ 4 ] - ( S3*sm[3] );
+		dm5 = sm[ 5 ] - ( S4*sm[4] );
+		dm6 = sm[ 6 ] - ( S5*sm[5] );
+		dm7 = sm[ 7 ] - ( S6*sm[6] );
+		dm8 = sm[ 8 ] - ( S7*sm[7] );
+		dm9 = sm[ 9 ] - ( S8*sm[8] );
+	}
+	dy0 = y.strides[ 0 ];
+
+	// Set the pointers to the first indexed elements in the respective ndarrays...
+	ix = x.offset;
+	im = mask.offset;
+	iy = y.offset;
+
+	// Cache references to the input, mask and output ndarray buffers...
```

Comment:
```suggestion
	// Cache references to the input, mask, and output ndarray buffers...
```

Use Oxford commas here and everywhere else.